src/app/lib/config/services/vietnamese.js

```diff
@@ -40,130 +40,133 @@ const service = {
+                'Chúng tôi có một số thay đổi quan trọng về Chính sách Riêng tư và Cookie, và muống bạn biết ý nghĩa của nó đối với bạn và dữ liệu của bạn.',
               linkText: null,
               last: null,
               linkUrl: null,
             },
           },
           accept: 'OK',
-          reject: "Find out what's changed",
+          reject: 'Tìm hiểu các thay đổi',
           rejectUrl: 'https://www.bbc.co.uk/usingthebbc/your-data-matters',
         },
         cookie: {
-          title: 'Let us know you agree to cookies',
+          title: 'Cho chúng tôi biết bạn đồng ý về cookie',
           description: {
             uk: {
-              first: 'We use ',
-              linkText: 'cookies',
+              first: 'Chúng tôi dùng ',
+              linkText: 'cookie',
               last:
-                ' to give you the best online experience. Please let us know if you agree to all of these cookies.',
+                ' để cho bạn trải nghiệm online tốt nhất. Cho chúng tôi biết bạn đồng ý về mọi cookie này.',
               linkUrl:
                 'https://www.bbc.co.uk/usingthebbc/cookies/what-do-i-need-to-know-about-cookies/',
             },
             international: {
-              first: 'We and our partners use technologies, such as ',
-              linkText: 'cookies',
+              first: 'Chúng tôi và các đối tác dùng công nghệ, như ',
+              linkText: 'cookie',
               last:
-                ', and collect browsing data to give you the best online experience and to personalise the content and advertising shown to you. Please let us know if you agree.',
+                ', và thu thập dữ liệu browsing để cho bạn trải nghiệm online tốt nhất và cá nhân hóa nội dung và quảng cáo cho bạn. Xin cho chúng tôi biết bạn có đồng ý không.',
               linkUrl:
                 'https://www.bbc.co.uk/usingthebbc/cookies/what-do-i-need-to-know-about-cookies/',
             },
           },
-          accept: 'Yes, I agree',
-          reject: 'No, take me to settings',
+          accept: 'Vâng, tôi đồng ý',
+          reject: 'Không, đưa tôi xem phần Settings',
           rejectUrl:
             'https://www.bbc.co.uk/usingthebbc/cookies/how-can-i-change-my-bbc-cookie-settings/',
         },
       },
       media: {
         audio: 'Âm thanh',
-        photogallery: 'Ảnh',
+        photogallery: 'Gallery hình ảnh',
         video: 'Video',
       },
     },
     brandSVG,
     mostRead: {
-      header: 'Most read',
-      lastUpdated: 'Last updated: ',
+      header: 'Đọc nhiều nhất',
+      lastUpdated: 'Cập nhật gần nhất: ',
     },
     footer: {
       externalLink: {
         href: 'https://www.bbc.co.uk/help/web/links/',
-        text: 'Read about our approach to external linking.',
+        text: 'Tìm hiểu cách chúng tôi tiếp cận việc dẫn tới trang ngoài',
       },
       links: [
         {
           href: 'https://www.bbc.com/news/help-41670342',
```

Comment:
There is a new link for this: https://www.bbc.com/vietnamese/institutional-49283563

Reply:
Done